framework/conda_build_config.yaml

```diff
@@ -1,5 +1,4 @@
 python:
-  - 3.6
```

Comment:
@peterfpeterson Does SNS need the Python 3.6 configuration at all?

Reply:
We were defaulting to python3.6 b/c it is still fairly common and the python3 version on RHEL7. It is probably a good question to ask a wider group. The impression I get (not measured in any way) is that people tend to build for 3.6 and 3.8 now.